src/Collection/Collection.router.ts

```diff
@@ -71,46 +67,30 @@ export class CollectionRouter extends Router {
   async getCollections(req: AuthRequest) {
     const eth_address = req.auth.ethAddress
 
-    const [dbCollections, remoteData] = await Promise.all([
-      Collection.findByEthAddressWithItems(eth_address),
-      collectionAPI.fetchCollectionsWithItemsByOwner(eth_address)
+    const [dbCollections, remoteCollections] = await Promise.all([
+      Collection.findByEthAddress(eth_address),
+      collectionAPI.fetchCollectionsByOwner(eth_address)
     ])
 
-    return new Bridge().consolidateCollections(
-      dbCollections,
-      remoteData.collections,
-      remoteData.items
-    )
+    return Bridge.consolidateCollections(dbCollections, remoteCollections)
   }
 
   async getCollection(req: AuthRequest) {
     const id = server.extractFromReq(req, 'id')
 
-    let dbCollection = await Collection.findOneWithItems(id)
-    let collection: CollectionWithItems | undefined
-
+    const dbCollection = await Collection.findOne<CollectionAttributes>(id)
     if (dbCollection) {
-      const remoteData = await collectionAPI.fetchCollectionWithItemsById(
+      const remoteCollection = await collectionAPI.fetchCollection(
         dbCollection.contract_address
       )
-      const collectionItems: CollectionItem[] = dbCollection.items.map(
-        item => ({ ...item, collection: dbCollection! })
-      )
-
-      collection = {
-        ...dbCollection,
-        items: new Bridge().consolidateItems(collectionItems, remoteData.items)
+      if (remoteCollection) {
+        return Bridge.mergeCollection(dbCollection, remoteCollection)
       }
 
-      if (dbCollection.contract_address) {
-        collection = {
-          ...collection,
-          ...remoteData.collection
-        }
-      }
+      return dbCollection
     }
 
-    return collection
+    return null
```

Comment:
I think we can return undefined here? to be consistent with other endpoints (like the item endpoint below, which just returns `dbItem: Item | undefined`)

Reply:
done 👍